lale/lib/lale/tee.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import lale.docstrings
+import lale.operators
+
+
+class _TeeImpl:
+    def __init__(self, observer=None):
```

Comment:
Here, "observer" means something different than in the "Observing" operator. Maybe use a different name, like "logger" or "listener"?

Reply:
changed to listener.